src/control/mod.rs

```diff
@@ -55,14 +55,19 @@ use std::iter::Zip;
 use std::mem;
 use std::num::NonZeroUsize;
 use std::ops::RangeBounds;
-use std::os::unix::io::{AsRawFd, RawFd};
+use std::os::unix::io::{AsFd, AsRawFd, RawFd};
 use std::time::Duration;
 
 use core::num::NonZeroU32;
 
+pub use nix::fcntl::OFlag;
+
 /// Raw handle for a drm resource
 pub type RawResourceHandle = NonZeroU32;
 
+/// Id of a Lease
+pub type DrmLeaseId = NonZeroU32;
```

Comment:
Can probably just call this `LeaseId` to be consistent.

The only types we have prefixed in this crate with `Drm` is types from the `drm_fourcc` crate.